trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_RHS_matrix.py

```diff
@@ -86,62 +86,68 @@ def compute_RHS_matrix(delta,phi,conditions,settings,geometry,propeller_wake_mod
 
     dt               = 0
     num_ctrl_pts     = len(aoa) # number of control points
-
+    num_eval_pts     = len(VD.XC)
     for network in geometry.networks:
         if propeller_wake_model:
+            # include the propeller wake effect on the wing
             if 'propellers' in network.keys(): 
-                if network.number_of_propeller_engines == None:
-                    pass
-                else:          
-                    # extract the propeller data structure
-                    identical_props = network.identical_propellers
-                    props           = network.propellers
-    
-                    # generate the geometry of the propeller helical wake
-                    wake_distribution, dt,time_steps,num_blades, num_radial_stations = generate_propeller_wake_distribution(props,identical_props,num_ctrl_pts,\
-                                                                                                                            VD,initial_timestep_offset,wake_development_time,\
-                                                                                                                            number_of_wake_timesteps,conditions)
-                    # compute the induced velocity
-                    prop_V_wake_ind = compute_wake_induced_velocity(wake_distribution,VD,num_ctrl_pts)
+                # extract the propeller wake and compute resulting induced velocities data structure
+                props           = network.propellers
+                prop_V_wake_ind = np.zeros((num_ctrl_pts,num_eval_pts,3))
+                
+                for p in props.keys():
+                    # check for wake method
+                    wake_method = props[p].Wake.wake_method
+                    
+                    if wake_method == "PVW":
```

Comment:
I would spell this out, keep acronyms for in function calculations

Reply:
changed to "Fidelity_One"